Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "NYTPhotoViewer",
+    platforms: [
+        .iOS(.v9)
+    ],
+    products: [
+        .library(
+            name: "NYTPhotoViewer",
+            targets: ["NYTPhotoViewer"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/pinterest/PINRemoteImage.git", from: "3.0.1")
+    ],
+    targets: [
+        .target(
+            name: "NYTPhotoViewer",
+            dependencies: ["PINRemoteImage"],
+            path: "NYTPhotoViewer",
+            exclude: [
+                "Info.plist"
+            ],
+            cSettings: [
+                .headerSearchPath("./include"),
```

Comment:
From the minimal objective-c SPM work I've done, I don't think this should be necessary.

Reply:
i tested, and you are right. c180675